ajax/libs/alloyeditor/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "alloyeditor",
+  "filename": "alloy-editor-all-min.js",
+  "version": "1.5.12",
```

Comment:
This version doesn't match the files included. Would it be easiest to just remove this attribute and remove the version files in this PR, so that the PR just adds the single package.json?